packages/app-check-types/index.d.ts

```diff
@@ -28,17 +28,35 @@ export interface FirebaseAppCheck {
     siteKeyOrProvider: string | AppCheckProvider,
     isTokenAutoRefreshEnabled?: boolean
   ): void;
+
+  /**
+   *
+   * @param isTokenAutoRefreshEnabled - If set to true, enables SDK to
+   * automatically refresh AppCheck token as needed. If false, disables this
```

Comment:
Is this a literal?  If not, maybe just App Check here and below (but is App Check actually publicly released right now?)

Reply:
No, it's not a literal, can change to App Check.

We are in a code freeze right now so the release will not happen until 5/11. The documentation can be published along with the rest on 5/13, see other comment.